lib/kanal/plugins/batteries_bridge/bridges/bridge.rb

```diff
@@ -11,6 +11,8 @@ module Bridges
         #
         class Bridge
           include Kanal::Core::Logging
+          # TODO: this line is obsolete, remove it
+          include Kanal::Core::Logging::Logger
```

Comment:
Please elaborate on TODO

Reply:
Done https://github.com/idchlife/kanal-plugins-batteries_bridge/pull/2/commits/763d9ab0d683494a15c3ed3e638ec2a560f593d6

Reply:
@notetell please elaborate here in the comments why there are 2 include statements.

We can use freely Kanal::Core::Logging::Logger without mentioning that there will be shortcut in the future.

Shortcut is not obligatory, it can be omitted in favour of fully specified path to the Logger mixin.

There are 2 things here now:

1) Why 2 includes?
2) We can use full path to logger mixin without mentioning future release of library-dependency if it does not have breaking changes.

Reply:
Including of Logging is indeed not needed, noted comment about Logger, removed TODO, will leave as is.